client/src/components/RelatedItems/OutfitItemsEntry.jsx

```diff
@@ -12,7 +12,7 @@ function OutfitItemsEntry({ img, item, handleRemove }) {
         alt={`Thumbnail for Outfit item ${item.name}`}
       />
       <Action onClick={() => { handleRemove(); }}>
-        Remove
+        X
```

Comment:
I add an X icon in one of my pending PRs that we can use here once it's merged to main.